### PR TITLE
Current Code for All network members

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**

### CODE REVIEW

File: organizations.json [hunk 1]
```diff
@@ -349,9 +345,8 @@
         "city": "Curitiba, PR",
         "latitude": "-25.4952",
         "longitude": "-49.2874",
-        "type": "Code for All",
         "tags": [
-            "Code for All"
+          "Brigade"
```
Comment (tdooner) [inline organizations.json]: We're doing a 4-space indent in this file. You can have all the formatting fixed for you automatically with:

```
bin/format-json organizations.json
```

You might need to `brew install jq moreutils` first before that script will run.

File: organizations.json [hunk 2]
```diff
@@ -667,9 +678,9 @@
         "city": "Accra, Ghana",
         "latitude": "5.5811",
         "longitude": "-0.1990",
-        "type": "Code for All",
         "tags": [
-            "Code for All"
+            "Code for All",
+            "Affiliate"
```
Comment (tdooner) [inline organizations.json]: Could you update the "tags" section in the README with descriptions of what "Affiliate" and "Fellowship" mean.

**Review by tdooner — COMMENTED** [review]
Great, thanks for helping keep this file up-to-date. Once Travis passes (it will if you run the `bin/format-json` script as described) we'll be good to merge.

**Comment by KrzysztofMadejski** [comment]
@tdooner done!

**Comment by tdooner** [comment]
Looks good, thanks @KrzysztofMadejski !

**Pull request merged**